src/components/UniversalNavbarExpanded/DropdownNav/DropdownChildren.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import get from 'lodash/get'
+
+// Reused assets from UniversalNavbar
+import { DropdownLinkIcon } from '../../UniversalNavbar/assets/icons.js'
+
+// EDS core components
+import { Layout, Footnote } from '../../index'
+
+// Parent component (UniversalNavbar) siblings
+import NavLink from '../NavLink'
+
+// DropdownChildren sibling components
+import DropdownCta from './DropdownCta'
+
+// Helpers
+import { removeLastWord, getLastWord } from '../../../helpers/splitLastWord'
+
+// Styles
+import styles from './DropdownChildren.module.scss'
+
+/**
+ * Content (links) displayed in the dropdown menu after hovering the parent element
+ *
+ * @param {string} extraClass - Extra top level class
+ * @param {object} child - CTA copy, url and link items with titles/urls
+ * @param {object} LinkComponent - Agnotistic Reach and React Router Link (ex. Gatsby's <Link>)
+ *
+ * @return {JSX.Element}
+ */
+const DropdownChildren = ({ extraClass, child, LinkComponent }) => {
+  // Split the links for the subnav into two columns
+  const columns = [[], []]
+  const childItems = get(child, 'subnav.items')
+
+  for (var i = 0; i < childItems.length; i++) {
+    if (i % 2 === 0) {
+      columns[0].push(childItems[i])
+    } else {
+      columns[1].push(childItems[i])
+    }
+  }
+
+  const classes = [styles.dropdownNavChildren]
+  if (extraClass) {
```

Comment:
Is there a way to pass in from `styles` rather than relying on a string className

Reply:
Some of the nesting complexity with :hover relied on this setup, but I'll see what I can do to revise.

Reply:
Leaving this one for now.